src/app.jsx

```diff
@@ -231,25 +231,50 @@ export const Application = () => {
                     : selected.length > 1 && selected.includes(selectedContext)
                     // eslint-disable-next-line max-len
                         ? [{ title: _("Copy"), onClick: _copyItem }, { title: _("Delete"), onClick: _deleteItem, className: "pf-m-danger" }]
-                        : [
-                            { title: _("Copy"), onClick: _copyItem },
-                            ...(selectedContext.type === "directory")
-                                ? [
-                                    {
-                                        title: _("Paste into directory"),
-                                        onClick: () => _pasteItem([...path, selectedContext.name], false),
-                                        isDisabled: clipboard === undefined
-                                    }
-                                ]
-                                : [],
-                            { type: "divider" },
-                            { title: _("Edit properties"), onClick: _editProperties },
-                            { title: _("Rename"), onClick: _renameItem },
-                            { type: "divider" },
-                            { title: _("Create link"), onClick: _createLink },
-                            { type: "divider" },
-                            { title: cockpit.format(_("Delete")), onClick: _deleteItem, className: "pf-m-danger" },
-                        ])
+                        : superuser.allowed
```

Comment:
This doesn't work, this needs to deal with UNIX permissions from the selected file. More details for example [here](https://www.redhat.com/sysadmin/linux-file-permissions-explained).

Reply:
What exactly doesn't work? Did I misunderstand the issue or implemented it wrongly?
I hid create link while in "limited access" mode

Reply:
The code needs to look at the current file permissions and the current logged in user, not if they are `sudo` or not.

I can only recommend researching how UNIX file permissions work in general https://linuxize.com/post/understanding-linux-file-permissions/

Reply:
Aha So every file should be worked on independently? but I should hide for example "create link" Like I did ?

Reply:
Yes, every file needs to be checked independent.